controllers/metrics/metrics.go

```diff
@@ -27,19 +27,18 @@ var (
 		Help:      "requests against the grafana api per instance",
 	}, []string{"instance_name", "method", "status"})
 
-	DashboardUrlRequests = prometheus.NewCounterVec(prometheus.CounterOpts{
+	ContentUrlRequests = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "grafana_operator",
-		Subsystem: "dashboards",
+		Subsystem: "content",
```

Comment:
We need to keep the old metric around as well, otherwise we're breaking compatibility with existing monitoring setups for the operator

Reply:
For simplicity, it should be enough to also increment `grafana_operator_dashboards_requests` during library panel operations. If more granularity is required, the new metric can be used

Reply:
ack - yeah, i wasn't sure what the policy was on breaking metrics - backwards compat is always good :) - i'll sort out something, it will be a teensy bit hacky, i think it's better long-term to have the metric more precisely communicate that it's for multiple resources now.

Reply:
ok, this should be resolved now. we use a type assertion to conditionally increment the old dashboard-specific metric.

btw i noticed that `GrafanaComApiRevisionRequests` was actually never registered in the prometheus registry! so i think that means it was never exposed, and thus we don't need to worry about breakage there.